docs/fio_distributed.md

```diff
@@ -173,6 +177,29 @@ jobfile configmap. These options will therefore override the global values for a
 Under most circumstances, the options provided in the EXPERT AREA here should not be modified. The `key=value`
 pairs under `params` here are used to append additional fio job options based on the job type. Each `jobname_match` in the list uses a "search string" to match a job name per `fio(1)`, and if a match is made, the `key=value` list items under `params` are appended to the `[job]` section of the fio jobfile configmap.
 
+## Dropping Rook-Ceph OSD Caches
+
+Dropping the OSD caches before workloads is a normal and advised part of tests that involve storage I/O.
+Doing this with Rook-Ceph requires a privileged pod running the same namespace as the Ceph pods and with
+the Ceph command tools available. To facilitate this, we provide the
+[resources/rook_ceph_drop_cache_pod.yaml](../resources/rook_ceph_drop_cache_pod.yaml) file, which will
+deploy a pod with the correct permissions and tools, as well as running a simple HTTP listener to trigger
+the cache drop by URL.
+**You must deploy this privileged pod in order for the drop caches requests in the workload to function.**
+
+```bash
+kubectl apply -f resources/rook_ceph_drop_cache_pod.yaml
+```
+
+*Note: If Ceph is in a namespace other than `rook-ceph` you will need to modify the provided YAML accordingly.*
```

Comment:
Which YAML do we update for the different ns? Looking at the provided YAML it is hard set to `rook-ceph`? If the YAML needs updating, the user running this work will have to rebuild the operator. Versus if you template the namespace and allow the user to override from the CR.